### PR TITLE
Make a test file for PropertiesFileOverridesReporter class

### PR DESCRIPTION
**Pull request by FeimeiChen**
# Ticket Link

[Ticket 1504](https://uhawaii.atlassian.net/browse/GROUPINGS-1504)

# List of squashed commits

- Commit 1: Make a test file for PropertiesFileOverridesReporter class
- Commit 2: edit the .gitignore file

# Test Checklist

- [x] Unit Tests Passed:
- [x] Integration Tests Passed:
- [x] General Visual Inspection:

### CODE REVIEW

File: src/test/java/edu/hawaii/its/api/configuration/PropertiesFileOverridesReporterTest.java
Comment (JorWo) [inline src/test/java/edu/hawaii/its/api/configuration/PropertiesFileOverridesReporterTest.java:1]: Same as your UI PR, reformat this file using our code style in IntelliJ by pressing `CTRL + ALT + l`

**Comment by fduckart** [comment]
Why do we need to update the .gitignore file?

**Comment by fduckart** [comment]
Maybe let's create a helper message to return the error message text, which we can test, then we can still declare and use the Logger in a conventional way.

**Comment by FeimeiChen** [comment]
> Why do we need to update the .gitignore file?

- Reason 
*-overrides.properties in .gitignore file, which ignores the dummy properties override file. so add !test-overrides.properties in .gitignore

- Question
  Could I remove the *-overrides.properties  from .gitignore instead of add !test-overrides.properties in .gitignore

File: src/test/java/edu/hawaii/its/api/configuration/PropertiesFileOverridesReporterTest.java
Comment (FeimeiChen) [reply src/test/java/edu/hawaii/its/api/configuration/PropertiesFileOverridesReporterTest.java:1]: done

**Review by JorWo — APPROVED** [review]
Looks great!

File: src/test/java/edu/hawaii/its/api/configuration/PropertiesFileOverrideReporterTest.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+@SpringBootTest(classes = { SpringBootWebApplication.class })
+@TestPropertySource(locations = "classpath:test-overrides.properties")
+public class PropertiesFileOverrideReporterTest {
+
+    @Autowired
+    private PropertiesFileOverrideReporter propertiesFileOverrideReporter;
+    private String expectedMessage;
```
Comment (fduckart) [inline src/test/java/edu/hawaii/its/api/configuration/PropertiesFileOverrideReporterTest.java:18]: You can remove that expectedMessage class level variable, I think.

Comment (FeimeiChen) [reply src/test/java/edu/hawaii/its/api/configuration/PropertiesFileOverrideReporterTest.java:18]: done, thank you.

**Pull request merged**